pac_windows.go

```diff
@@ -0,0 +1,91 @@
+package ieproxy
+
+import (
+	"strings"
+	"syscall"
+	"unsafe"
+)
+
+type WINHTTP_AUTOPROXY_OPTIONS struct {
```

Comment:
>Should the WINHTTP_AUTOPROXY_* constants also have a lowercase letter prefix?

Yes. If you don't want to change the name, please move those into internal/syscall/pac_windows.go